src/components/StoryPointButton.tsx

```diff
@@ -0,0 +1,13 @@
+export default function StoryPointButton({
+	storyPoint,
+}: { storyPoint: number | string }) {
+	return (
+		<input
+			type='radio'
+			aria-label='radio'
```

Comment:
bug: Why did you add this `aria-label` with value "radio"? The aria-label is the same a `<label>`. Ask yourself: If it had a visually label, what would be the label? Probably `{storyPoint} points`, right?

Reply:
I learned how to make the radio buttons into buttons, and and the example in DaisyUI had the aria-label='radio'. I don't know much about aria-label other than it used for accessibility, but that's about it. 

I'll change that to your suggestion. That makes sense!

Reply:
DONE